openbas-model/src/main/java/io/openbas/database/repository/TeamRepository.java

```diff
@@ -28,6 +28,9 @@ public interface TeamRepository extends CrudRepository<Team, String>,
   @NotNull
   Optional<Team> findByName(@NotNull final String name);
 
+  @NotNull
+  List<Team> findAllByName(@NotNull final String name);
+
```

Comment:
Should we use ignoreCase, or would it be better to create a task to rearrange this from the creation?

Reply:
Good question, what do you think @jborozco & @EllynBsc ?
Could we have two teams like that: TeamA & tEAMa ?
Or not ?

Reply:
Adding ignoreCase